CONTRIBUTING.md

```diff
@@ -57,26 +56,40 @@ Now run:
 
 And your Fauxton dev server will be up and running at `localhost:8000`.
 
+Caveats:
```

Comment:
Weird. I haven't worked on Fauxton in awhile but this shouldn't be required. Fauxton dev mode runs through a local proxy. It just expects a CouchDB instance at `localhost:8000`

Reply:
In the past, I had the same problem when working with attachments. I don't know if its still the case

Reply:
@garrensmith @popojargo When I start dev this time around it clears out the CORS configuration set by the plugin and the CORS config in the UI is inaccessible:

<img width="1054" alt="Screen Shot 2019-07-10 at 7 28 55 PM" src="https://user-images.githubusercontent.com/1408766/61011571-17dfe500-a349-11e9-93a8-a54debca2e36.png">